package.json

```diff
@@ -23,7 +23,7 @@
     "test:build:dist": "npx jest --selectProjects \"Gulp tasks\" --testMatch \"**/*build-dist*\"",
     "lint": "npm run lint:js && npm run lint:scss",
     "lint:js": "eslint --cache --cache-location .cache/eslint --cache-strategy content --color --ignore-path .gitignore \"**/*.{cjs,js,mjs}\"",
-    "lint:scss": "stylelint \"app/**/*.scss\" \"src/**/*.scss\""
+    "lint:scss": "stylelint --cache --cache-location .cache/stylelint --color --ignore-path .gitignore \"app/**/*.scss\" \"src/**/*.scss\""
```

Comment:
If we changed the stylelint config, will we need to clear the cache to force it to re-lint all files against the new config? I can't see anything in [the stylelint docs](https://github.com/stylelint/stylelint/blob/main/docs/user-guide/usage/options.md#cache) to suggest this is handled for us.

(Same applies for eslint)

Reply:
ESLint automatically marks the cache as stale when the config changes

Let me check what StyleLint does (good shout)